zhaquirks/owon/Owon_PC321_Z.py

```diff
@@ -0,0 +1,94 @@
+"""Quirk for Owon PC321 Z."""
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import Basic, Identify
+from zigpy.zcl.clusters.smartenergy import Metering
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.owon import Owon
+
+
+class Owon_PC321_Z_Simple_Metering(CustomCluster, Metering):
+    """Owon PC321 CustomCluster"""
+
+    ep_attribute: str = "smartenergy_metering"
```

Comment:
I think this isn't needed, because the `ep_attribute` is already set in `Metering` (which we're inheriting here)?